docs/tool/js/core/housing-insights.js

```diff
@@ -296,6 +296,11 @@ var setSubs = controller.controlSubs.setSubs,
 
  //TODO should these be wrapped up into a 'helpers' namespace? Or maybe the new utils section?
 
+// HELPER roundTo
```

Comment:
For this and the comma-separated thing, I realized that D3 already provides this functionality - we should use the d3.format() function to do this work for us. https://github.com/d3/d3-format